io.liri.BaseApp.yaml

```diff
@@ -0,0 +1,48 @@
+id: io.liri.BaseApp
+branch: 5.11
+runtime: org.kde.Platform
+sdk: org.kde.Sdk
+runtime-version: 5.11
+separate-locales: false
+modules:
+- name: qbs
+  sources:
+  - type: archive
+    url: http://download.qt.io/official_releases/qbs/1.11.0/qbs-src-1.11.0.tar.gz
+    sha256: 6ce30c468e688f6843468324a34a7191409b471c3ff6f3e834ef8bfa7b3467cd
+  - type: patch
+    path: patches/qbs-fileinfo-exists.patch
+  - type: script
+    dest-filename: configure
+    commands:
+    - qmake QBS_INSTALL_PREFIX=/app CONFIG+=qbs_enable_project_file_updates qbs.pro
+- name: qbs-shared
+  buildsystem: simple
+  build-commands:
+  - ./qbs-configure
+  - ./qbs-build qbs.buildVariant:debug modules.qbs.installRoot:/ project.prefix:/app
+  sources:
+  - type: archive
+    url: https://github.com/lirios/qbs-shared/releases/download/v1.2.0/liri-qbs-shared-1.2.0.tar.xz
+    sha256: 1cbf3475bef23f3533bb164ad9eda7042d488a0e7a6a954b48cfb4c8cee1d1aa
+  - type: file
+    path: sources/qbs-configure
+    dest-filename: qbs-configure
+  - type: file
+    path: sources/qbs-build
+    dest-filename: qbs-build
+- name: fluid
+  buildsystem: simple
+  build-commands:
+  - ./qbs-configure
+  - ./qbs-build qbs.buildVariant:debug modules.qbs.installRoot:/ modules.lirideployment.prefix:/app project.useSystemQbsShared:true project.withDocumentation:false
+  sources:
+  - type: archive
+    url: https://github.com/lirios/fluid/releases/download/v1.0.0/fluid-1.0.0.tar.xz
+    sha256: c6cec8f91c8ab5619f6b4b61204b9e6fa6865c9915206a49f8ae522d45514a65
+  - type: file
```

Comment:
Oh BTW I wonder if I should install these in the base app since each and every app is going to use them to build.

Reply:
Personally I don't love moving build commands, especially one or two liners, to their own scripts at all.

Reply:
Done